### PR TITLE
Improve output for check script

### PR DESCRIPTION
**Pull request by rahearn**
# 🎫 Addresses issue: #5 

## 🛠 Summary of changes

* Don't swallow output from `compliance --check`
* Change how assessment-results.json output is handled
* Upload those results to the evidence repo

### CODE REVIEW

File: bin/plant-helper [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env bash
+
+usage="
+$0: Plant external evidence in the evidence locker
```
Comment (zjrgov) [inline bin/plant-helper:4]: thought: This is enjoyable phrasing.

File: bin/prune-helper [hunk 2]
```diff
@@ -5,34 +5,32 @@ $0: Prune obsolete evidence from the evidence locker
...
 
 Options:
 -h: show help and exit
--c: prune config. ex: '{\"path/to/filename.json\":\"Reason it is being pruned\"}'
+-f: file to prune. Should be given as a relative path from the root of the evidence repo
+-r: Reason the evidence is being pruned
 -l: https version of locker repository
 -b: main branch used in locker repository. Default: 'main'
--e: your email address. Defaults to '$GIT_EMAIL'
 -d: Dry run mode
```
Comment (zjrgov) [inline bin/prune-helper]: suggestion (non-blocking): should this change to `-n` to be consistent with `plant-helper`?

File: bin/check [hunk 3]
```diff
@@ -4,8 +4,15 @@ set -e
...
+if [ "$1" = "" ]; then
+  ar="$(mktemp -d)/auditree.json"
+else
+  ar="$1"
+fi
+
+python c2p/result_to_compliance.py -c cdef.json -i /tmp/compliance/check_results.json > $ar
+
+locker_repo=`jq -r ".locker.repo_url" auditree/auditree.json`
+plant-helper -f "$ar" -c assessment-results -d "Auditree check assessment-results" -l "$locker_repo" -n
```
Comment (zjrgov) [inline bin/check]: question (non-blocking): why dry-run mode?

**Review by zjrgov — DISMISSED** [review]

File: bin/check (hunk 3 above)
Comment (rahearn) [reply bin/check]: oops - it shouldn't be anymore

**Review by zjrgov — APPROVED** [review]

**Pull request merged**